ocs_samples/basic_samples/SDS/JavaScript/Angular/readme.md

```diff
@@ -114,13 +119,11 @@ To use SDS, you define SdsTypes that describe the kinds of data you want
 to store in SdsStreams. SdsTypes are the model that define SdsStreams.
 SdsTypes can define simple atomic types, such as integers, floats, or
 strings, or they can define complex types by grouping other SdsTypes. For
-more information about SdsTypes, refer to the SDS
-documentation <https://ocs-docs.osisoft.com/Documentation/SequentialDataStore/Data_Store_and_SDS.html>.
+more information about SdsTypes, refer to the [SDS documentation](https://ocs-docs.osisoft.com/Documentation/SequentialDataStore/Data_Store_and_SDS.html).
 
 In the sample code, the SdsType representing WaveData is defined in the buildWaveDataType method of
 datasrc.component.ts. WaveData contains properties of integer and double atomic types. 
-The constructions begins by defining a base SdsType for each atomic type and then defining
-Properties of those atomic types.
+The constructions begins by defining a base SdsType for each atomic type.
```

Comment:
I don't think you're responsible for this one, but I think it should be "the construction begins"